consent-ws/src/main/java/org/broadinstitute/consent/http/service/DataAccessParser.java

```diff
@@ -197,6 +202,17 @@ public PDAcroForm fillDARForm(Document dar, Map<String, String> researcherProper
         return acroForm;
     }
 
+    private Map<String, String> parseDatasetDetail(Document dar) {
+        ArrayList<Document> datasetDetail = (ArrayList<Document>) dar.get(DarConstants.DATASET_DETAIL);
+        Map<String, String> datasetDetailMap = new HashMap<>();
+        datasetDetail.forEach((doc) -> {
+            String objectId = doc.getString(DarConstants.OBJECT_ID) != null ? doc.getString(DarConstants.OBJECT_ID) : "";
+            datasetDetailMap.put("dataSetName", doc.getString("name"));
+            datasetDetailMap.put("objectId" ,objectId);
```

Comment:
Similar to comment above ... potential here for putting an empty string as a key in `datasetDetailMap` which seems brittle.
Also spacing:
```suggestion
            datasetDetailMap.put("objectId", objectId);
```